de_docs/getting-started/user-guides/dcrwallet-tickets.md

```diff
@@ -0,0 +1,208 @@
+# **Buying Tickets With dcrwallet**
+
+Last updated for v0.8.2 on 4/18/2017
```

Comment:
I like the idea of a last updated for version, but I don't think we should have dates in the last updated by.  Better to just tie things to software versions.